mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeans.scala

```diff
@@ -482,12 +482,15 @@ object KMeans {
   /**
    * Trains a k-means model using the given set of parameters.
    *
-   * @param data training points stored as `RDD[Vector]`
-   * @param k number of clusters
-   * @param maxIterations max number of iterations
-   * @param runs number of parallel runs, defaults to 1. The best model is returned.
-   * @param initializationMode initialization model, either "random" or "k-means||" (default).
-   * @param seed random seed value for cluster initialization
+   * @param data Train with a `RDD[Vector]` of data points.
```

Comment:
* `a RDD` -> `an RDD`. Please also check other occurrence in this PR.
* `data` should be described as a noun. So the previous version looks better to me.